src/iaaft.jl

```diff
@@ -166,15 +269,15 @@ function iaaft_iters(ts::AbstractArray{T, 1} where T;
         # series, but sorted according to the new spectrum. The shuffled series
         # is generated by taking the inverse Fourier transform of the spectrum
         # consisting of the original amplitudes, but having randomised phases.
-        surrogate[:] = real(ifft(spectrum)) # ifft normalises by default
-        surrogate[sortperm(surrogate)] = original_sorted # map original values onto shuffle
-        push!(surrogates, surrogate[:])
+        surr[:] = real(ifft(spectrum)) # ifft normalises by default
```

Comment:
Why are you using `ifft` here instead of `irfft`? It is more performant, actually exactly 2x more performant. See how I've done it in `RandomFourier`.

Reply:
I do use `irfft` in the new API implementation, just haven't changed anything but cosmetics in the old `iaaft`.

Reply:
AH sorrY! I should have been more careful in my review :)